packages/gafl-webapp-service/src/handlers/page-handler.js

```diff
@@ -26,6 +27,21 @@ const getBackReference = async (request) => {
   return status.backRef.previous
 }
 
+const performTracking = async (request, path) => {
+  if ([CONTACT_SUMMARY.uri, LICENCE_SUMMARY.uri].includes(path)) {
```

Comment:
Should only send a product view when on the licence summary, not on the contact summary